source/_integrations/devolo_home_control.markdown

```diff
@@ -33,7 +33,18 @@ ha_integration_type: hub
 
 {% include integrations/config_flow.md %}
 
-Please do not change the URL provided in the advanced mode unless you know what you are doing.
+{% configuration_basic %}
+Email / mydevolo ID:
+  description: "Email address you used to register the central unit at mydevolo."
+Password:
+  description: "Password of your mydevolo account."
+mydevolo URL:
+  description: "URL of the mydevolo stage to use. Please do not change the URL unless you know what you are doing."
```

Comment:
Then why are why exposing this even?

Reply:
Nothing changed in my opinion on the matter though:

> IMHO, it doesn't belong here at all.
> If an option isn't valuable for the larger part of the users, it should not be added.

From an Home Assistant perspective, this is a stronger vision we have now; as our user base is growing and changing. We should consider removing it.

Reply:
Ok, I will follow that. Can we merge this one here and https://github.com/home-assistant/core/pull/132703 first, so that I can start with a clean state?

Reply:
I think we shouldn't. That would mean we add translations, that the community will start to translate, and will remove afterwards? That seems like unneeded waste of time for our translators.

Reply:
Ah, that's reasonable. I didn't think of translations. Let me then think of how to not mix different topic in the same MR, handle different branches and still keep track :-)